### ISSUE TITLE
Make `yarn setup` log into Firebase and deploy your Firestore rules

### ISSUE THREAD

**Issue by bfirsh** [boxwise/boxwise#255]
The reason I thought of this is that `yarn setup` makes you log in with Firebase, then the next step in the getting started guide makes you do that again, which is a bit silly.

We can probably roll all this into `yarn setup` somehow so it gets done automatically:

```
$ yarn run firebase login
$ yarn run firebase use --add
$ yarn run deploy-firestore
```

See also #239

**Comment by rubenspgcavalcante** [comment]
Maybe a flag like `yarn setup --deploy`. As the setup script create a local configuration file, will not be fine if everytime you run, do a deployment.

**Comment by HaGuesto** [comment]
Closed ("Paused") due to discussion #275.
